compose/config/validation.py

```diff
@@ -121,15 +122,16 @@ def anglicize_validator(validator):
     return 'a ' + validator
 
 
+def _parse_key_from_error_msg(error):
```

Comment:
This doesn't belong out here, it's a private internal function for `process_errors` and not to be used outside of it.